src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.css

```diff
@@ -6,8 +6,7 @@
 }
 
 h1, h2, h3, h4, h5 {
-    /*font-family: 'Segoe UI',Tahoma,Arial,Helvetica,sans-serif;*/
-    font-weight: 100;
+    font-weight: 400;
```

Comment:
Out of curiosity, what if you just leave this out entirely (and the one below)? Sometimes the problem is that *too many* things are specified.

Reply:
This screen shot looks pretty nice to me. Maybe we give that a go?